packages/snaps-execution-environments/scripts/build.js

```diff
@@ -6,6 +6,7 @@ const browserify = require('browserify');
 const { promises: fs } = require('fs');
 const LavaMoatBrowserify = require('lavamoat-browserify');
 const path = require('path');
+const { Worker, workerData, isMainThread } = require('worker_threads');
```

Comment:
I don't love the `isMainThread` pattern, so I hope we can fix this some other way 🤔